notebooks/triposr-3d-reconstruction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q wheel setuptools pip --upgrade
</pre><p>why is it done after majority of dependencies installed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>